standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -498,7 +505,32 @@ Each term in this vocabulary has a human-readable description and may include ad
 * Data-property <nowiki>{extensionAllowed}[Something that pertains to data or information.]</nowiki>
 ** Data-artifact <nowiki>[An anomalous, interfering, or distorting signal originating from a source other than the item being studied.]</nowiki>
 *** Biological-artifact <nowiki>[A data artifact arising from a biological entity being measured.]</nowiki>
+**** Eye-artifact <nowiki>[Artifact from eyes.]</nowiki> 

```

Comment:
All of the items in this section are self-defining -- could people contribute suggestions about descriptions.

Should all of the items in this section start  "Data artifact ..." rather than "Artifact ..."?